QtWebApp/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ if(CMAKE_BUILD_TYPE MATCHES Debug)
 		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
 	else()
 		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -fsanitize=undefined")
+                set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -fsanitize=undefined")
```

Comment:
Can you try to align the indentation with the line above it please

Reply:
Of course. Glorious mix of tabs and spaces, didn't see that in the terminal.
qtwebapp is the only repo I currently deal with where tabs are used, sorry for that.